src/Illuminate/Validation/ValidationRuleParser.php

```diff
@@ -195,11 +195,9 @@ public static function parse($rules)
             return [$rules, []];
         }
 
-        if (is_array($rules)) {
-            $rules = static::parseArrayRule($rules);
-        } else {
-            $rules = static::parseStringRule($rules);
-        }
+        $rules = is_array($rules)
```

Comment:
How is this any better than the IFs? I find `if(){ ... }else{ ... }` much clearer.